engine/apps/base/models/live_setting.py

```diff
@@ -80,14 +82,24 @@ class LiveSetting(models.Model):
             "after you update them."
         ),
         "TWILIO_ACCOUNT_SID": (
-            "Twilio username to allow amixr send sms and make phone calls, "
+            "Twilio account SID/username to allow OnCall to send SMSes and make phone calls, see "
             "<a href='https://support.twilio.com/hc/en-us/articles/223136027-Auth-Tokens-and-How-to-Change-Them' target='_blank'>"
-            "more info</a>."
+            "here</a> for more info. Required."
+        ),
+        "TWILIO_API_KEY_SID": (
+            "Twilio API key SID/username to allow OnCall to send SMSes and make phone calls, see"
+            "<a href='https://www.twilio.com/docs/iam/keys/api-key' target='_blank'>"
+            "here</a> for more info. Either (TWILIO_API_KEY_SID + TWILIO_API_KEY_SECRET) or TWILIO_AUTH_TOKEN is required."
+        ),
+        "TWILIO_API_KEY_SECRET": (
+            "Twilio API key secret/password to allow OnCall to send SMSes and make phone calls, see"
+            "<a href='https://www.twilio.com/docs/iam/keys/api-key' target='_blank'>"
+            "here</a> for more info. Either (TWILIO_API_KEY_SID + TWILIO_API_KEY_SECRET) or TWILIO_AUTH_TOKEN is required."
         ),
         "TWILIO_AUTH_TOKEN": (
-            "Twilio password to allow amixr send sms and make calls, "
+            "Twilio password to allow OnCall to send SMSes and make calls, "
             "<a href='https://support.twilio.com/hc/en-us/articles/223136027-Auth-Tokens-and-How-to-Change-Them' target='_blank'>"
-            "more info</a>."
+            "here</a> for more info. Either (TWILIO_API_KEY_SID + TWILIO_API_KEY_SECRET) or TWILIO_AUTH_TOKEN is required."
         ),
         "TWILIO_NUMBER": (
```

Comment:
I pushed [d0b8512](https://github.com/grafana/oncall/pull/643/commits/d0b851299267a8e1e091ee9205bfcb88fb222ec0) for some minor grammar changes.

Nitpick comment but maybe consider extracting some of the wording in `TWILIO_ACCOUNT_SID`, `TWILIO_API_KEY_SID`, `TWILIO_API_KEY_SECRET`, and `TWILIO_AUTH_TOKEN` into a variable.